contrib/inventory/azure_rm.py

```diff
@@ -634,8 +695,19 @@ def _get_powerstate(self, resource_group, name):
         except Exception as exc:
             sys.exit("Error: fetching instanceview for host {0} - {1}".format(name, str(exc)))
 
+        return self._extract_power_state(vm.instance_view)
+
+    def _get_scaleset_vm_powerstate(self, resource_group, scaleset, instance_id):
+        try:
+            instance_view = self._compute_client.virtual_machine_scale_set_vms.get_instance_view(resource_group, scaleset, instance_id)
+        except Exception as exc:
+            sys.exit("Error: fetching instance_view for scaleset host {0}/{1} - {2}".format(scaleset, instance_id, str(exc)))
+
+        return self._extract_power_state(instance_view)
+
+    def _extract_power_state(self, instance_view):
         return next((s.code.replace('PowerState/', '')
-                    for s in vm.instance_view.statuses if s.code.startswith('PowerState')), None)
+                     for s in instance_view.statuses if s.code.startswith('PowerState')), None)
```

Comment:
This change is correct and follows pep8 style guidelines. Indentation does not always have to be an increment of 4 if it aligns with the above structure.